acquia_lift.install

```diff
@@ -639,6 +638,186 @@ function acquia_lift_update_7017() {
 }
 
 /**
+ * Miscellaneous changes required for the new workflow, includes converting legacy
+ * campaigns to "personalizations".
+ */
+function acquia_lift_update_7018() {
+  // The queue is now gone so delete the queue class variable.
+  variable_del('queue_class_acquia_lift_sync');
+
+  // Get all old-style agents and convert them where possible.
+  $result = db_query("SELECT a.machine_name, a.label, o.plugin, o.osid, o.decision_name, o.options, o.targeting FROM {personalize_agent} a INNER JOIN {personalize_option_sets} o ON o.agent = a.machine_name WHERE a.plugin IN ('acquia_lift', 'acquia_lift_simple_ab')");
```

Comment:
I thought you were only doing acquia_lift agents... I think Shay was looking into the AB agents but I'm sure he'd be happy to know he doesn't have to.